app/assets/stylesheets/admin.scss

```diff
@@ -650,3 +650,9 @@ ul.settings-menu {
     }
   }
 }
+
+.merge {
+  color: red;
+  font-weight: bold;
+  text-transform: uppercase;
+}
```

Comment:
We don't actually need the red, I should have noted in the requirements that it was red just to call attention to the new link.  So in other words, the merge link should look like the others in that menu bar.